ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -19,9 +19,27 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 gentoo_git_repo: https://github.com/gentoo/gentoo.git
 # Select a specific commit in the gentoo_git_repo that should be used for the bootstrap,
 # e.g. by checking: https://github.com/gentoo/gentoo/commits/master
-gentoo_git_commit: 7eaa2512d1e6ddb44e3b41bbddf6c74723f234ce
+# 2021.12
+# gentoo_git_commit: 7eaa2512d1e6ddb44e3b41bbddf6c74723f234ce
+# 2022.10 (Oct 3 2022) considered
+# gentoo_git_commit: 063b3e0e3d31acd88f453831f71d74d976d66359
+# 2022.10 (Oct 3 2022) considered 2
+#   one commit before gcc-config-2.6 was added (by cc6a27ec99c1e08ac51c69ff0ab4c2b8a5578e2e)
+#     the below commit is 3200 commits behind 7eaa2512d1e6ddb44e3b41bbddf6c74723f234ce (2021.12)
+#     079738bd484ed8ecd83e2a82a1bbff97b5b004e2 is from Sep 23, 2022
+# gentoo_git_commit: 079738bd484ed8ecd83e2a82a1bbff97b5b004e2
+# 2022.10 (Oct 11 2022) actually used
+# gentoo_git_commit: 3e6e7da40a59134ab2afee9b27b5b0c2051f21a4
+# 2022.11 (Oct 28 2022) first try
+#   this installs gcc-10.4.1_p20221006 in bootstrap stage2/3
+#   subsequently 10.4.0 cannot be set as default gcc (via gcc-config)
+#   trying to set 10.4.1_p20221006 fails
+#   we mask sys-devel/gcc below to not install anything newer than 10.4.0
+# gentoo_git_commit: c2d8ce0e1b6206a225a9f2547bbc65c79218756c
+# 2022.11 (Nov 3 2022) second iteration made for PR
```

Comment:
@trz42 I'm not sure we should keep this whole history, but the problems with the patched 10.4.1 versions would be good to highlight somehow. I think sticking to 10.4.0 makes sense.

The 10.4.1 versions are test builds I think... Maybe @amadio can clarify?

Reply:
We can prune the comments. For me they add some context right now while we got going again. If we update more frequently or even build compat layers with a bot, they might not provide much useful context or there are other ways to document the history.

Reply:
Pruned some comments.

Reply:
GCC ebuilds with `_p` in the version are versions of GCC with extra patches by Gentoo, usually to solve bugs, so you can consider them similar to the other versions. For example sys-devel/gcc-13.0.0_pre20221030 was added to fix https://bugs.gentoo.org/879049. Also, the only reason that GCC 10.x is used for bootstrapping is because GCC 11 requires C++11 to build, and that itself requires a new enough compiler that may not be available on old systems. You don't have to stick to older versions of GCC in EESSI. You can just leave the masks free and choose GCC by adding a slotted version to your set, like `sys-devel/gcc:11` if you want GCC 11.x but don't care which minor version. It's quite safe to keep it updated within the same major version (i.e. that won't break packages built on top of the compat layer).